zerver/lib/error_notify.py

```diff
@@ -23,8 +23,8 @@ def logger_repr(report: Dict[str, Any]) -> str:
     return "Logger {logger_name}, from module {log_module} line {log_lineno}:".format(**report)
 
 def user_info_str(report: Dict[str, Any]) -> str:
-    if report['user_full_name'] and report['user_email']:
-        user_info = "{user_full_name} ({user_email})".format(**report)
+    if report['user_full_name'] and report['user_email'] and report['user_role']:
```

Comment:
I was a bit unsure if I needed to add this `report['user_role']` check because I actually think only one of those checks on user information is needed. If there is anyone of them, there will be a value for the others because they are set in a sequence on `logging_handlers.py`. But I guess it doesn't hurt to check.

Reply:
Yeah, I think it'd be cleaner to refactor this to just have a `user` object and have its set of fields be always the same.